app/models/order_item.rb

```diff
@@ -40,7 +40,7 @@ def set_defaults
   end
 
   def update_message(level, message)
-    self.updated_at = DateTime.now
+    self.updated_at = Time.zone.now
```

Comment:
@syncrou This should be in GMT correct?

Reply:
It will be if we set the rails app config default time to UTC
That's what the `zone` method does.

Reply:
Though it may be that by default.

Reply:
Yeah, default is UTC - http://patrikonrails.com/posts/2

Reply:
@bzwei @syncrou This is creating a child object the parent will not get automatically updated.